### DISCUSSION TITLE
i am new here.so happy to meet everyone.i from china.i program about 10years

### DISCUSSION THREAD

**Discussion by showhand168** [community/community#147203] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

go go

**Comment by shinybrightstar** [comment]
👋 Welcome to the GitHub Community on Discussions, @showhand168 ! Happy to have you here 🙂


To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).
